lib/Aftok/Currency/Zcash.hs

```diff
@@ -7,6 +7,7 @@ module Aftok.Currency.Zcash
     Z._IVK,
     RPCError (..),
     ZValidateAddressErr (..),
+    ZcashConfig (..),
     ZcashdConfig (..),
```

Comment:
If we don't plan to run zcashd, should this dependency be removed, or is that independent of the actual service?

Reply:
Yeah, maybe it should just be gotten rid of, or moved out to a separate library.